tests/RobotFramework/tests/tedge_agent/http_server/entity_store_api.robot

```diff
@@ -126,6 +127,28 @@ Delete entity tree
     ...    {"@topic-id":"device/child2//","@parent":"device/main//","@type":"child-device"}
     ...    ${entities}
 
+Entity twin update apis
+    ${entity}=    Execute Command
+    ...    curl -X PUT http://localhost:8000/tedge/entity-store/v1/entities/device/main///twin/maintenance_window -H 'Content-Type: application/json' -d '5'
+    ${entity}=    Evaluate    ${entity}
+    Should Be Equal    ${entity["maintenance_window"]}    ${5}
```

Comment:
Robotframework has a keyword to do this rather than using the expression evaluation, `${5}`.

```suggestion
    Should Be Equal As Integers    ${entity["maintenance_window"]}    5
```

Reply:
Resolved by f837935c